src/redux/reducers/componentConfig/initialState.js

```diff
@@ -1,3 +1,46 @@
+const embeddingPreviewInitialConfig = {
+  width: 600,
+  height: 500,
+  pointSize: 5,
+  toggleInvert: '#FFFFFF',
+  masterColour: '#000000',
+  umap1Domain: null,
+  umap2Domain: null,
+  plotToDraw: true,
+  titleText: '',
+  titleSize: 20,
+  titleAnchor: 'start',
+  axisTitlesize: 13,
+  axisTicks: 13,
+  transGrid: 0,
+  axesOffset: 10,
+  masterFont: 'sans-serif',
+  xAxisText: '',
+  yAxisText: '',
+  pointStyle: 'circle',
+  pointOpa: 5,
+  g1Color: 'red',
+  g2mColor: 'green',
+  sColor: 'blue',
+  legendTextColor: '#FFFFFF',
+  legendEnabled: true,
+  legend: null,
+  legendPosition: 'top-right',
+  geneexpLegendloc: '',
+  colGradient: 'spectral',
+  labelSize: 28,
+  labelShow: 1,
+  labelFont: 2,
+  labelsEnabled: true,
+  reverseCbar: false,
+  selectedClusters: [],
+  testVar: null,
+  bounceX: 0,
+  plotTitle: 'default clusters',
+  maxHeight: 600,
+  maxWidth: 700,
+};
+
```

Comment:
I believe this change is extraneous at this point, we have merged the other MR in that uses the same config/schema as the plots and tables.

Reply:
Oh yea :S this comes up as I used off a previously unmerged branch of the "refactor the embedding preview ticket". As git diffs, it doesn't detect this change. I'll remove it in next commit